packages/core/content-manager/server/src/history/controllers/history-version.ts

```diff
@@ -68,13 +68,28 @@ const createHistoryVersionController = ({ strapi }: { strapi: Core.Strapi }) =>
         ...getValidPagination({ page: params.page, pageSize: params.pageSize }),
       });
 
+      const sanitizedResults = await async.map(results, async (version: any) => {
```

Comment:
Why do we need any? results does have a type, I'm surprised it's not inferred

Reply:
I'm not sure maybe @Marc-Roig knows how the inference/typings could work with the async.map

Reply:
We are using a curried version of pMap, for some reason the lodash/fp `curry` function doesn't work too well on types :sad-pepe:

Reply:
Bummer, well I can at least replace the any here with a type from the contract for `version`.